src/Types.jl

```diff
@@ -264,6 +264,16 @@ struct EnvCache
 end
 EnvCache() = EnvCache(get(ENV, "JULIA_ENV", nothing))
 
+function write_env_usage(manifest_file::AbstractString)
+    !ispath(logdir()) && mkpath(logdir())
+    usage_file = joinpath(logdir(), "usage.toml")
+    touch(usage_file)
+    !isfile(manifest_file) && return
+    open(usage_file, "a") do io
+        TOML.print(io, Dict(manifest_file => [Dict("time" => now())]))
```

Comment:
For this kind of function I would recommend just printing out the content you want instead of bothering with using the TOML print function. I.e. just do something like this:
```jl
println(io, "[[$(tomlkey(manifest_file))]]")
println(io, "time = ", repr(time))
println(io, "command = ", repr(command))
```
It'll be faster, more reliable and the code is easier to understand. It's not like the TOML format is going to radically change how it prints this kind of information.

Reply:
I updated, hopefully it is clearer. Had to use the `format` to get the `Z` in the end.